### DISCUSSION TITLE
Why is the drillthrough widget not showing columns from joined table?

### DISCUSSION THREAD

**Discussion by tht-activeviam** [atoti/atoti#684] [Q&A] [answered]
Atoti version `0.6.6`

When I join the table using their non-key field, they do not appear in Drillthrough widget.

Notebook to reproduce the issue => https://github.com/tht-activeviam/perso-tht-atoti-playground/blob/main/notebooks/drillthrough%20Demo.ipynb

**Comment by tibdex** [comment, marked as answer]
In your notebook, the join between the **Products** and **Categories** tables is partial. Thus, the columns of **Categories** do not end up in the "selection" and can indeed not be used in a drillthrough.

**Reply by tht-activeviam** [reply]
Hi @tibdex ,
Even with "partial" join, the cube is still loading properly but the drillthrough. 
Are we officially saying that we do not support partial join? I don't think such thing mentioned in the official documentation.

**Reply by tibdex** [reply]
> Are we officially saying that we do not support partial join?

Partial joins (also called Copper joins in ActivePivot) are supported but the columns of partially joined tables do not end up in what is called the cube "selection" and thus cannot be used in drillthroughs.
